### PR TITLE
Updated goodbye world example so that it matches the hello world example

### PR DESCRIPTION
**Pull request by mpetason**
# Changes

- Updates the goodbye world example so that it matches the hello world example and outputs the correct text
- Also updated the output from hello goodbye to goodbye world

### CODE REVIEW

**Comment by mpetason** [comment]
Working on getting the CLA updated.

File: content/en/docs/Getting Started/pipelines.md [hunk 1]
```diff
@@ -32,9 +32,10 @@ spec:
   steps:
     - name: goodbye
       image: ubuntu
-      script: |
-        set -e
-        echo "Goodbye World!"
+      command:
```
Comment (afrittoli) [inline content/en/docs/Getting Started/pipelines.md:35]: The version we want to recommend to users is actually the `script` one, so I would update other docs to match that

File: content/en/docs/Getting Started/pipelines.md [hunk 2]
```diff
@@ -93,7 +94,7 @@ It may take a few moments before your `Task` completes. When it executes, it sho
 show the following output:
 
 ```
-[goodbye] Hello Goodbye!
+[goodbye] Goodbye World!
```
Comment (afrittoli) [inline content/en/docs/Getting Started/pipelines.md:97]: Nice :)

File: content/en/docs/Getting Started/pipelines.md (hunk 1 above)
Comment (mpetason) [reply content/en/docs/Getting Started/pipelines.md]: Got it, reverted back to a script and updated it so that we get the correct output.

**Comment by avinashsingh6** [comment]
/lgtm 

Thanks for these changes.

**Pull request merged**